src/reference/informative/informC08_interpretation_of_mathematics3.rst

```diff
@@ -0,0 +1,164 @@
+.. _informC08_interpretation_of_mathematics3:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding how the units work
+
+    We know that units in real life make a big difference, but in the CellML specification universe, they don't. 
+    Each :code:`variable` element has to specify a corresponding :code:`units` element, but this combination doesn't affect the validity of the :code:`math` element.
+    It does, however, make a difference to how your model needs to be interpreted.
+    Some examples are shown below.
```

Comment:
The reason for all of this is that the CellML spec authors (as far as I can tell) would _like_ all models to have consistent units, but many historically important ones (e.g. in cell electrophysiology) do not, and we don't want to restrict the usefulness of CellML to models written by people who also care about units.

So maybe just say that units are there to help users, and _can_ be used to check for errors

Reply:
d07f1f3 ... but not sure if I've got it right here either!

Reply:
Cool! Maybe add a note + explicit examples to say that

"x in [V] connected to y in [A]" --> Spec says not valid
"x in [V] connected to y in [mV]" --> Spec says y = 1000[mV/V] * x 
"x in [V] = 3 [A]" --> Spec says is valid
"x in [V]  =1 [mV]" --> Spec says "x = 1" (units not taken into account here)

Reply:
2d94c68